contracts/interfaces/ISwap.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface ISwap {
+    struct Offer {
+        // 32 byte slot 1
+        address seller;
+        bool isOpen;
+        uint88 offeredTokenDecimals;
+        // 32 byte slot 2
+        address offeredToken;
+        uint96 minPrice;
+        // 32 byte slot 3
+        address biddingToken;
+        uint96 minBidSize;
+        // 32 byte slot 4
+        uint128 totalSize;
+        uint128 availableSize;
+        // 32 byte slot 5
+        uint128 highestPrice;
```

Comment:
Why is `highestPrice` required for storage? We can infer the average price from `totalSales` right? We can get the highestPrice through off-chain means if need be.

Reply:
Average should be fine